tests/features/ui/ui_admin.feature

```diff
@@ -1,17 +1,20 @@
 @ui
 Feature: Admin UI
-
     Scenario Outline: Admin page view by admin users
+        technicaladmin is an admin user as role: rodsadmin
+        functionaladminpriv is an admin user in group: priv-admin
+
```

Comment:
Example of how we can change the comments so that it won't break my syntax highlighter. [More info](https://cucumber.io/docs/gherkin/reference/) (specifically the Descriptions) header.